lib/checkPullRequestLabels.js

```diff
@@ -215,15 +216,6 @@ module.exports.checkChangelogLabel = async function (context) {
 
   const isCollaborator = await isUserCollaborator(context, userName);
   if (!isCollaborator) {
-    // Sleep for 5 seconds and refetch the pull request so it
-    // contains all the requested reviewers.
-    await sleep(5000);
-    console.log('Finished sleeping');
-    const pullRequestResponse = await context.github.pulls.get(
-      context.repo({ pull_number: pullRequestNumber })
-    );
-    pullRequest = pullRequestResponse.data;
-
```

Comment:
Why is this removed?

Reply:
It is no longer required as this didn't fix the issue before.